nemo/collections/asr/metrics/rnnt_wer.py

```diff
@@ -326,6 +339,32 @@ def decode_ids_to_tokens(self, tokens: List[int]) -> List[str]:
         """
         raise NotImplementedError()
 
+    def update_joint_fused_batch_size(self):
+        if self.joint_fused_batch_size is None:
+            # do nothing and let the Joint itself handle setting up of the fused batch
+            return
+
+        if not hasattr(self.decoding.joint, 'set_fused_batch_size'):
```

Comment:
Do we have a joint decoder which does not have set_fused_batch_size?

Reply:
Not yet.. but in the future we may have (say we use a different joint impl for some reason). Just planning for a future that may come or not.